src/App.tsx

```diff
@@ -393,7 +393,11 @@ export default function App() {
                   </div>
                 </Slide>
               ) : (
-                <NavBar organization={selectedOrganization} setShowNavBar={setShowNavBar} />
+                <NavBar
+                  organization={selectedOrganization}
+                  setShowNavBar={setShowNavBar}
+                  backgroundTransparent={location.pathname === APP_PATHS.HOME}
```

Comment:
what to capture both `/home` and `/home/`?

Reply:
changed it to `location.pathname.startsWith(APP_PATHS.HOME)`

Reply:
Nice, that works 👍